devtools/api-mock/contacts/data.json

```diff
@@ -16,7 +16,9 @@
   "deleted": 0,
   "pi": { "technic": 87, "context": 45, "comportment": 25 },
   "tags": ["Robots", "Planet Express"],
-  "infos": {},
+  "infos": {
+    "birthday": "2016-05-09T15:01:42.11600"
```

Comment:
I don't know how this information is going to be used, but it might be a good idea to store/retrieve it as `YYYY-MM-DD` `YYYY` or `MM-DD`

It can be parsed by JavaScript `Date` constructor anyway